### ISSUE TITLE
Quibit cell not charging ender io items

### ISSUE THREAD

**Issue by scalda** [joekaiser/QuantumFlux#116]
Submitting a bug report? Please fill out the form below the best you can.

*Version:* latest for 1.10.2

*Mod pack:* Hermit pack 1.4.0

*Summary:* quibit cell not charging ender io items

____

####What is happening? i think this is due to ender io using the forge system of power now instead of rf

####How do you think it should be? implement the forge unit of power for charging

####How can I reproduce the problem? have any ender io item and the quibit cell in your inventory

**Comment by joekaiser** [comment]
Have you heard if the Quibit Clusters are able to provide power to EIO machines?

**Comment by scalda** [comment]
yes they work to power the machines same for the rf exciters, it seems just the items and dark steel armor doesn't charge from the quibit cell

**Comment by Yadokingu** [comment]
This might be an enderIO problem though. The quibit cell will charge battaries from Actually Editions, but both won't charge enderIO items.

**Comment by scalda** [comment]
Enderio uses the forge unit energy system now not any RF api. Even thou the
machines and wires can use RF the mod isn't coded with a RF api. Where as
actually additions uses a RF api not the FU energy yet
